src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -142,8 +168,14 @@ trait SharedFileSystem {
       name -> adjusted
     }
 
-    // If this call is using Docker, adjust input paths, otherwise return unaltered input paths.
-    if (call.docker.isDefined) inputs map adjustPath else inputs
+    inputs map adjustPath
+  }
+
+  private def hashPath(originalPath: Path, size: Int = 8): String = {
+    val hash = MessageDigest.getInstance("MD5").digest(originalPath.toAbsolutePath.getParent.toString.getBytes) map { byte =>
+      f"$byte%02x"
+    } mkString ""
```

Comment:
You're just lucky I left my pitchfork at home.  But with some tricks to shorten this up you can get rid of the local and maybe the line length wouldn't be too bad?

```scala
/* somewhere else */
lazy val MD5Digester = MessageDisgest.getInstance("MD5")
/* here */
MD5Digester.digest(...) map { b => f"$b%02x" } mkString take(size)
```